src/sentry_check.py

```diff
@@ -40,33 +38,18 @@ def visit_ExceptHandler(self, node):
         self.generic_visit(node)
 
     def visit_UAdd(self, node):
-        trailing_nodes = list(map(type, self.node_window[-4:]))  # NOQA: B315
+        trailing_nodes = list(map(type, self.node_window[-4:]))
         if trailing_nodes == [ast.UnaryOp, ast.UAdd, ast.UnaryOp, ast.UAdd]:
             originator = self.node_window[-4]
             self.errors.append(B002(originator.lineno, originator.col_offset))
         self.generic_visit(node)
 
     def visit_ImportFrom(self, node):
-        if node.module in B307.names:
-            self.errors.append(B307(node.lineno, node.col_offset))
-
-        if node.module == "__future__":
-            for nameproxy in node.names:
-                if nameproxy.name == "absolute_import":
-                    self.has_absolute_import = True
-                    break
-
-        if node.module == "sentry.utils.compat":
-            for nameproxy in node.names:
-                if nameproxy.name in B315.names:
-                    self.satisfies_B315_imports = True
-                    break
-
-        if node.module == "itertools":
-            for nameproxy in node.names:
-                if nameproxy.name in B316.names:
-                    self.itertools_izip = True
-                    break
+        # if node.module == "__future__":
+        #     for nameproxy in node.names:
+        #         if nameproxy.name == "absolute_import":
+        #             self.has_absolute_import = True
+        #             break
```

Comment:
Delete?

Reply:
Nah, I plan on following up using some of this to implement __future__ disallow.